### PR TITLE
Additions to blogpost: Running ownCloud with Caddy

### PR DESCRIPTION
**Pull request by DenBeke**
Some additions to #13

### CODE REVIEW

File: public/blog/caddy_and_owncloud.md [hunk 1]
```diff
@@ -88,6 +90,11 @@ If you need previews for videos and documents you also need to install the follo
 - ffmpeg or avconv
 - LibreOffice
 
+
+For optimal integration with Caddy, accepting PHP-FPM request on a TCP socket instead of a Unix socket is better:  
+Replace `listen = /run/php/php7.0-fpm.soc` to `listen = 127.0.0.1:9000` in `/etc/php/7.0/fpm/pool.d/www.conf`.
```
Comment (DenBeke) [inline public/blog/caddy_and_owncloud.md]: @peteruithoven: this thing :)
(answering on your question of #13)

Comment (peteruithoven) [inline public/blog/caddy_and_owncloud.md]: That helps! 
But now I'm getting `File not found.`

Comment (peteruithoven) [inline public/blog/caddy_and_owncloud.md]: access.log mentions
```
17/Mar/2016:16:10:21 +0000 [ERROR 404 /phpinfo.php] Primary script unknown
172.17.0.1 - [17/Mar/2016:16:10:21 +0000] "GET /phpinfo.php HTTP/1.1" 404 16
```

Comment (DenBeke) [inline public/blog/caddy_and_owncloud.md]: Which is a typical docker-with-php thing :p 
But way out of scope of this pull requests... Maybe you should comment on my [blogpost](https://denbeke.be/blog/webdevelopment/serving-owncloud-with-caddy/), to keep these PR "spam" free :)

Comment (peteruithoven) [inline public/blog/caddy_and_owncloud.md]: For sure, will do.

**Comment by mholt** [comment]
Thanks for the updates!

**Pull request merged**